app/lib/cob_alma/requests.rb

```diff
@@ -161,6 +172,24 @@ def self.physical_material_type(items_list)
       material_types.uniq.compact
     end
 
+    def self.physical_material_type_and_descriptions(items_list)
+      types_and_descriptions = items_list.all.map { |item|
+        Hash[item.physical_material_type["desc"], [item.description]] unless item.physical_material_type["value"] == ""
+      }.uniq
```

Comment:
```suggestion
      }.uniq.compact
```

@sensei100 without `.compact` at the end this can produce `[nil]` which will cause an error below